docs/.gitignore

```diff
@@ -0,0 +1,91 @@
+# Logs
```

Comment:
Let's get this into master asap

Reply:
```
npm i -g gitignore
gitignore node
```

You'll thank me later.

Also this is specific to the docs directory, not at the root.